docs/docsource/requirements.txt

```diff
@@ -0,0 +1,103 @@
+#
```

Comment:
What's the difference between this file and https://github.com/scality/Zenko/blob/45b856ac80dbda1c9a42a31dc0c6f5da322f5883/docs/requirements.txt?

Are they the same? should we remove the one on the `docs` folder?